stories/__tests__/SQFormInclusionList.stories.test.tsx

```diff
@@ -140,4 +140,97 @@ describe('SQFormInclusionList Tests', () => {
       expect(submitButton).toBeEnabled();
     });
   });
+
+  it("It should display the error message 'friends field must have at least 5 items' when less than 5 items are selected in the fieldArray", async () => {
+    render(<WithValidation />);
+
+    const submitButton = screen.getByRole('button', {name: /submit/i});
+
+    await waitFor(() => {
+      const errorText = screen.getByText(
+        /friends field must have at least 5 items/i
+      );
+      expect(errorText).toBeInTheDocument();
+    });
+
+    await waitFor(() => {
+      expect(submitButton).toBeDisabled();
+    });
+
+    // get one of the unchecked checkbox elements
+    const jaipal = screen.getByLabelText('Jaipal');
+
+    //click checkbox to make validation pass by adding a 5th item to the array
+    userEvent.click(jaipal);
+
+    //check that our validation message is displaying 'All fields completed'
+    await waitFor(() => {
+      const validText = screen.getByText(/All fields completed/i);
+      expect(validText).toBeInTheDocument();
+    });
+
+    // check to ensure submit button is properly enabled
+    await waitFor(() => {
+      expect(submitButton).toBeEnabled();
+    });
+
+    // uncheck a checkbox to make the validation fail (less than 5 boxes checked)
+    const joe = screen.getByLabelText('Joe');
+    userEvent.click(joe);
+
+    await waitFor(() => {
+      const errorText = screen.getByText(
+        /friends field must have at least 5 items/i
+      );
+      expect(errorText).toBeInTheDocument();
+    });
+
+    const jane = screen.getByLabelText('Jane');
+    userEvent.click(jane);
+
+    // get error message element
+    await waitFor(() => {
+      const errorText = screen.getByText(
+        /friends field must have at least 5 items/i
+      );
+      expect(errorText).toBeInTheDocument();
+    });
+
+    // finally, expect submit button to be disabled
+    await waitFor(() => {
+      expect(submitButton).toBeDisabled();
+    });
+  });
+
+  it("It should display the error message 'friends field must have at least 5 items' when our form initially starts in a invalid state and it should continue to display the same error message after unchecking one of the already checked checkboxes", async () => {
```

Comment:
The test description says `it` so we don't need to begin our description with `it` as well.

I agree with testing the initial state of the form but I don't think we need to test again that unchecking another item keeps us in an error state.

So we could shorten this description down to something like 

`should display the error message "friends field must have at least 5 items" when our form is initially loaded with less than 5 items checked`

Reply:
ahh ok! I went ahead and pushed up a commit removing the extra "it" word and removing the additional test for unchecking another item keeping us in the error state.